apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -27,6 +27,7 @@ export const sageAccountWebThemeOverrides: ThemeOptions = {
             '0px 4px 4px rgba(0, 0, 0, 0.03), 0px 2px 2px rgba(0, 0, 0, 0.03), 0px 1px 1px rgba(0, 0, 0, 0.03)',
           borderRadius: '3px',
           fontWeight: 700,
+          fontSize: '16px',
```

Comment:
Align fontSize of outlined button with contained button, e.g. "Cancel" and "Save" buttons in the ConfirmationDialog. Consider whether other button variants (e.g. text, secondaryContained) should also use the same fontSize (or if a default value should be set)

Reply:
They probably all should. I think some of the original modifications to this theme were more heavy-handed than they should have been.

Reply:
Sure thing, done in the next commit. 

We may also want to consider updating some Links to Buttons (or adjusting the font size of links), so that the font size matches. For example, see the AccountSettings page, where the "Change Password" button font size is 16px, but the "More Information" link is 14px

<img width="763" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/d9ae7933-971c-40cd-94c9-5176b27f72c0">